### PR TITLE
Pongo 2.0

### PR DESCRIPTION
**Pull request by hanshuebner**
See https://github.com/Kong/kong-pongo/issues/322 for breaking changes

### CODE REVIEW

**Comment by Tieske** [comment]
maybe we should remove only the listed versions, and leave the artifacts in, just to keep this PR managable. After the merge we can update the artifacts.

The failures in CI; multi-dict, iirc that was the reason to add the python-dev package.

**Comment by Tieske** [comment]
the `python-dev` package isn't solving this. Seems to need python 3.7 at least, and requires the wheel thing updates, see https://httpie.io/docs/cli/universal.

File: pongo.sh [hunk 1]
```diff
@@ -1153,7 +1153,7 @@ function main {
     compose run --rm --use-aliases \
       -e KONG_LICENSE_DATA \
       -e KONG_TEST_DONT_CLEAN \
-      -e LD_LIBRARY_PATH=/kong-plugin \
+      -e LD_LIBRARY_PATH=/kong-plugin:/usr/local/kong/lib \
```
Comment (Tieske) [inline pongo.sh:1156]: any specific reason for this change? (just trying to understand why)

Comment (hanshuebner) [reply pongo.sh:1156]: Yes, sorry, should have been more verbose.  The XML shared libraries that are now bundled with Kong are in `/usr/local/kong/lib`, which is not on the standard system search path for shared libraries.    This is why :)

**Pull request merged**